api/RTCCertificate.json

```diff
@@ -143,54 +143,6 @@
             "deprecated": false
           }
         }
-      },
-      "getSupportedAlgorithms": {
```

Comment:
Confirmed not in Chromium, Gecko, or WebKit, just a FIXME in WebKit:
https://github.com/WebKit/WebKit/blob/751a4873a15c12356e413dc947dd29d481889a2c/Source/WebCore/Modules/mediastream/RTCCertificate.idl#L45